addons/web_settings_dashboard/static/src/js/dashboard.js

```diff
@@ -76,12 +76,37 @@ var DashboardInvitations = Widget.extend({
         'click .o_web_settings_dashboard_access_rights': 'on_access_rights_clicked',
         'click .o_web_settings_dashboard_user': 'on_user_clicked',
         'click .o_web_settings_dashboard_more': 'on_more',
+        'focusout .select2-input': 'on_focusout',
     },
     init: function(parent, data){
         this.data = data;
         this.parent = parent;
         return this._super.apply(this, arguments);
     },
+    start: function(){
+        var temp = this._super.apply(this, arguments);
+        var self = this;
+        this.s2input = this.parent.$('textarea#user_emails').select2({
+            placeholder: _t("Enter e-mail addresses"),
+            tags: [],
+            tokenSeparators: [" ", ","],
+            dropdownCss: {display:'none'},
+            width: '100%',
+        });
+        this.s2input.on("select2-open", function(e){
+            $('.select2-drop-mask').remove();
+        });
```

Comment:
Why is this handler needed ? I removed it and couldn't find a difference.

Reply:
Actually removing this handler will produce the aforesaid bug.